core/src/main/resources/urlfilters.json

```diff
@@ -4,7 +4,7 @@
       "class": "com.digitalpebble.storm.crawler.filtering.depth.MaxDepthFilter",
       "name": "MaxDepthFilter",
       "params": {
-        "maxDepth": 0
+        "maxDepth": 10
```

Comment:
That's unrelated to the functionality of this PR and can/should be overridden in the user config. It should not be changed here.